README.md

```diff
@@ -397,6 +394,132 @@ const { hash } = await erc165SessionKeyProvider.sendUserOperation({
+        },
+        {
+            condition: ParamCondition.EQUAL,
+            offset: 0,
+            param: pad(<SPECIFIC_ADDRESS>, { size: 32 }),
+        },
+        ],
+    }
+]
+
+// 4. Initialize SessionKey Validator Provider
+const sessionKeyProvider = await SessionKeyProvider.init({
+      projectId, //ZeroDevProject
+      sessionKey, // Session Key Signer
+      sessionKeyData: {
+        validAfter: 0,
+        validUntil: 0,
+        permissions,
+        paymaster, // Paymaster Address : zeroAddress means accept userOp without paymaster, oneAddress means reject userOp without paymaster, other address means accept userOp with paymaster with the address
+      },
+      opts: {
+        accountConfig: {
+          accountAddress,
+        },
+        validatorConfig: {
+          mode: ValidatorMode.plugin,
+          executor: zeroAddress,
+          selector,
+        },
+      },
+});
+
+// 5. Get the validator instance
+// If the session key is generated in the client and need to be approved from the server
+// create an EmptyValidator instance and send it to the server
+let validator = await EmptyValidator.fromValidator(sessionKeyProvider.getValidator());
+// OR else just get the validator instance from the SessionKeyProvider
+validator = sessionKeyProvider.getValidator();
+
+// 6. Get enable signature from default ECDSA validator provider
+const enableSig = await ecdsaProvider
+    .getValidator()
+    .approveExecutor(
+        accountAddress,
+        executeSelector,
+        zeroAddress,
+        0,
+        0,
+        validator
+    );
+
+// 7. Set enable sig in SessionKey Validator Provider
+sessionKeyProvider.getValidator().setEnableSignature(enableSig);
+
+// 8. Send the transaction
+const { hash } = await sessionKeyProvider.sendUserOperation({
+          target: ERC20Address,
+          data: encodeFunctionData({
+                    abi: TEST_ERC20Abi,
+                    functionName: "transfer",
+                    args: ["RECIPIENT_ADDRESS", "AMOUNT_TO_TRANSFER"],
+                }),
+});
+
+
+// Creating session key on the server
+// After step 7. above do following server-side and send the sessionData to the client
+const sessionData = sessionKeyProvider
+      .getValidator()
+      .serializeSessionData(<SESSION_PRIVATE_KEY>);
+
+// On client side
+const sessionDataClient =
+    SessionKeyValidator.deserializeSessionData(sessionData);
+const sessionKeyProvider = await SessionKeyProvider.init({
+      projectId, //ZeroDevProject
+      sessionKey: ZeroDevLocalAccountSigner.privateKeyToAccountSigner(sessionDataClient.sessionPrivateKey),
+      sessionKeyData: sessionDataClient.sessionKeyData,
+      opts: {
+        accountConfig: {
+          accountAddress,
+        },
+        validatorConfig: {
+          mode: ValidatorMode.plugin,
+          executor: zeroAddress,
+          selector,
+          enableSignature: sessionDataClient.enableSignature
+        },
+      },
```

Comment:
Same as above -- this part is too complicated.  With the old session key I believe the account address itself is serialized as a part of `sessionData` so there's no need for the client to manually specify the account address.

Reply:
On it.